autospec/pkg_integrity.py

```diff
@@ -477,7 +479,8 @@ def verify(self, recursion=False):
         if sign_isvalid(self.package_sign_path) is False:
             self.print_result(False, err_msg='{} is not a GPG signature'.format(self.package_sign_path))
             try:
-                os.unlink(self.package_sign_path)
+                #os.unlink(self.package_sign_path)
```

Comment:
Any reason to keep the try and code commented out? Might as well just print and return.

Reply:
That was for debugging so it would stop pulling the rug out from under me :) Restored the unlink so it will still happen when the signature really fails.